DancingGoat/Helpers/SampleDeliveryClient.cs

```diff
@@ -0,0 +1,160 @@
+            return _deliveryClient.GetItemsAsync<T>(p);
+        }
+
+        public Task<DeliveryItemListingResponse<T>> GetItemsAsync<T>(IEnumerable<IQueryParameter> parameters)
+        {
+            var p = parameters.Concat(_sampleSiteFilter);
+            return _deliveryClient.GetItemsAsync<T>(p);
+        }
+
+        public Task<JObject> GetTypeJsonAsync(string codename)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<JObject> GetTypesJsonAsync(params string[] parameters)
+        {
+            parameters.ToList().Add(_sampleSiteFilter.FirstOrDefault().GetQueryStringParameter());
+            return _deliveryClient.GetTypesJsonAsync(parameters);
+        }
+
+        public Task<ContentType> GetTypeAsync(string codename)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<DeliveryTypeListingResponse> GetTypesAsync(params IQueryParameter[] parameters)
+        {
+            var p = parameters.Concat(_sampleSiteFilter);
+            return _deliveryClient.GetTypesAsync(p);
+        }
+
+        public Task<DeliveryTypeListingResponse> GetTypesAsync(IEnumerable<IQueryParameter> parameters)
+        {
+            var p = parameters.Concat(_sampleSiteFilter);
+            return _deliveryClient.GetTypesAsync(p);
+        }
+
+        public Task<ContentElement> GetContentElementAsync(string contentTypeCodename, string contentElementCodename)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<JObject> GetTaxonomyJsonAsync(string codename)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<JObject> GetTaxonomiesJsonAsync(params string[] parameters)
+        {
+            parameters.ToList().Add(_sampleSiteFilter.FirstOrDefault().GetQueryStringParameter());
+            return _deliveryClient.GetItemsJsonAsync(parameters);
+        }
+
+        public async Task<TaxonomyGroup> GetTaxonomyAsync(string codename)
+        {
+            var taxonomies = await GetTaxonomiesAsync(_sampleSiteFilter);
+            return taxonomies.Taxonomies.FirstOrDefault(x => x.System.Codename == Brewer.ProductStatusCodename);
+        }
+
+        public Task<DeliveryTaxonomyListingResponse> GetTaxonomiesAsync(params IQueryParameter[] parameters)
+        {
+            var p = parameters.Concat(_sampleSiteFilter);
+            return _deliveryClient.GetTaxonomiesAsync(p);
+        }
+
+        public async Task<DeliveryTaxonomyListingResponse> GetTaxonomiesAsync(IEnumerable<IQueryParameter> parameters)
+        {
+            var p = parameters.Concat(_sampleSiteFilter);
+            return await _deliveryClient.GetTaxonomiesAsync(p);
+        }
+
+        public IContentLinkUrlResolver ContentLinkUrlResolver { get; set; }
+        public ICodeFirstModelProvider CodeFirstModelProvider { get; set; }
+        public IInlineContentItemsProcessor InlineContentItemsProcessor { get; }

```

Comment:
These props should mirror the respective ones in _deliveryClient as they do in LanguageClient